### PR TITLE
CLDR-16122 v43 spec: fix bad link, update wording

### PR DESCRIPTION
**Pull request by srl295**
CLDR-16122

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by richgillam — DISMISSED** [review]
LGTM

File: docs/ldml/tr35.md [hunk 1]
```diff
@@ -3188,8 +3188,8 @@ Of course, both of these techniques can be applied.
 
 ### <a name="Vertical_Slicing" href="#Vertical_Slicing">Vertical Slicing</a>
 
-Locales chosen to filter out depend very much on the particular implementation.
-Some information that might be useful for determining that that is the [Supplemental Territory Information](https://unicode.org/reports/tr35/tr35-info.html#Supplemental_Territory_Information),
+The choice of locales to include depends very much upon particular implementations.
+Some information which might be useful for determining that that is the [Supplemental Territory Information](tr35-info.md#Supplemental_Territory_Information),
```
Comment (macchiati) [inline docs/ldml/tr35.md:3192]: Should be
Some information which might be useful for determining that that is the
=>
Some information that might be useful for determining the choice is found in the

**Review by macchiati — CHANGES_REQUESTED** [review]
Thanks! 

However, these are not the only problems I pointed to.

We shouldn't ever have:
https://unicode.org/reports/tr35/tr35-.*/.(html|md)?#
or 
https://unicode.org/reports/tr35/#
or variants of these.

That is, we should always have **relative** links within the parts of the spec, except for the (few) instances that must point to specific versions.
----

tr35.md (9 matches)
3,192: Some information that might be useful for determining that that is the [Supplemental Territory Information](https://unicode.org/reports/tr35/tr35-info.html#Supplemental_Territory_Information), 
tr35-general.md (2 matches)
1,269: > The `from` attribute consists of a sequence of elements. Each element matches one character and may consist of a codepoint or a UnicodeSet (both as defined in [UTS#35 section 5.3.3](https://www.unicode.org/reports/tr35/#Unicode_Sets)). 
tr35-personNames.md (4 matches)
104: This document provides the [LDML](http://www.unicode.org/reports/tr35/) specification for formatting of personal names, using data, structure, and examples. 
262: * `#PCDATA `is a space delimited list of one or more [unicode_locale_id](https://unicode.org/reports/tr35/#unicode_locale_id)s. Normally each locale is limited to language, script, and region. The _und_ locale ID may only occur once, either in _surnameFirst_ or _givenFirst_, but not both, and matches all base locales not explicitly listed. 
543: These nameOrderLocales will match any locale with a zh or ja [unicode_language_subtag](https://unicode.org/reports/tr35/#unicode_language_subtag) and any locale with a CN, TW, SG, HK MO, HU, or JP [unicode_region_subtag](https://unicode.org/reports/tr35/#unicode_region_subtag).

File: docs/ldml/tr35.md (hunk 1 above)
Comment (srl295) [reply docs/ldml/tr35.md:3192]: > Thanks!
> 
> However, these are not the only problems I pointed to.
> 
> We shouldn't ever have: [https://unicode.org/reports/tr35/tr35-.*/.(html|md)?#](https://unicode.org/reports/tr35/tr35-.*/.(html%7Cmd)?#) or https://unicode.org/reports/tr35/# or variants of these.
> 
> ## That is, we should always have **relative** links within the parts of the spec, except for the (few) instances that must point to specific versions.
> tr35.md (9 matches) 3,192: Some information that might be useful for determining that that is the [Supplemental Territory Information](https://unicode.org/reports/tr35/tr35-info.html#Supplemental_Territory_Information), tr35-general.md (2 matches) 1,269: > The `from` attribute consists of a sequence of elements. Each element matches one character and may consist of a codepoint or a UnicodeSet (both as defined in [UTS#35 section 5.3.3](https://www.unicode.org/reports/tr35/#Unicode_Sets)). tr35-personNames.md (4 matches) 104: This document provides the [LDML](http://www.unicode.org/reports/tr35/) specification for formatting of personal names, using data, structure, and examples. 262: * `#PCDATA `is a space delimited list of one or more [unicode_locale_id](https://unicode.org/reports/tr35/#unicode_locale_id)s. Normally each locale is limited to language, script, and region. The _und_ locale ID may only occur once, either in _surnameFirst_ or _givenFirst_, but not both, and matches all base locales not explicitly listed. 543: These nameOrderLocales will match any locale with a zh or ja [unicode_language_subtag](https://unicode.org/reports/tr35/#unicode_language_subtag) and any locale with a CN, TW, SG, HK MO, HU, or JP [unicode_region_subtag](https://unicode.org/reports/tr35/#unicode_region_subtag).

will work on the match.  I thought i hit all of these except for ones which pointed to specific versions.

Comment (srl295) [reply docs/ldml/tr35.md:3192]: OK. i definitely was not looking for `https://unicode.org/reports/tr35/#`  - that's an odd form.

Comment (srl295) [reply docs/ldml/tr35.md]: fixed

**Comment by srl295** [comment]
I added another regex to catch the unversioned `reports/tr35/#` case.

**Comment by macchiati** [comment]
Does it also catch .../tr35# ?

**Comment by srl295** [comment]
> Does it also catch .../tr35# ?

yes (as noted above)

**Review by macchiati — APPROVED** [review]

**Pull request merged**